Tasks/Podvornaya/Task3/Task1.py

```diff
@@ -0,0 +1,25 @@
+string = input("Type your string here:")
+el_num = int(input("How much repeating symbols do you want to see?"))
+
+count_dict = {}
+
+for i in string:
+    count = 0
+    for j in string:
+        if j == i:
+            count +=1
+            continue
```

Comment:
kinda overcomplicated, you could've just store this intermediate values in the dictionary itself